lib/sensu/api/routes/checks.rb

```diff
@@ -22,6 +22,26 @@ def get_check
             not_found!
           end
         end
+
+        # DELETE /checks/:check_name
+        def delete_check
+          # history:*:check_name
+          # history:*:check_name:last_ok
+          # result:*:check_name
+          check_name = parse_uri(CHECK_URI).first
+          @redis.keys("result:*:#{check_name}") do |result_keys|
```

Comment:
@amdprophet we can use `smembers clients` to compose the result keys, avoid this use of `keys`.

Reply:
clients.map {|client_name| "result:#{client_name}:#{check_name}"}